src/__tests__/hooks.test.js

```diff
@@ -491,8 +487,8 @@ describe.skip('hooks', () => {
   it('can read tracking data from props.tracking.getTrackingData()', () => {
     const mockReader = jest.fn();
 
-    const TestOptions = ({ onProps, children }) => {
-      const { Track } = useTracking({ onProps, ...testDataContext });
+    const TestOptions = ({ onProps, child, children }) => {
+      const { Track } = useTracking({ onProps, child, ...testDataContext });
```

Comment:
@tizmagik Not sure whether this was what you were intending, but this was what I needed to do to match the expected test output.